### ISSUE TITLE
Browser support?

### ISSUE THREAD

**Issue by awlayton** [OADA/oada-cache#9]
Is it supposed to work in browser? Because it does not for me. At least not in electron.

**Comment by awlayton** [comment]
I get `Error: A dynamic link library (DLL) initialization routine failed` because it tries to use leveldb like in regular node.

**Comment by abalmos** [comment]
I believe that is pouchdb using leveldb (and I assume it uses IndexedDB in
browser). Maybe it has trouble detecting the environment in electron?

Andrew

**Comment by awlayton** [comment]
Yes pouch is doing it. But there are ways to make pouch not try to load it in browser. Perhaps it works as is in a regular browser and is just confused by electron. I know pouch _can_ work in electron without trying to load leveldown.
